wolfssl/internal.h

```diff
@@ -4468,6 +4482,12 @@ struct WOLFSSL {
         void* RsaEncCtx;      /* Rsa Public  Encrypt   Callback Context */
         void* RsaDecCtx;      /* Rsa Private Decrypt   Callback Context */
     #endif /* NO_RSA */
+    void* GenPreMasterCtx;   /* Generate Premaster Callback Context */
+    void* GenMasterCtx;      /* Generate Master Callback Context */
+    void* GenSessionKeyCtx;  /* Generate Sesssion Key Callback Context */
+    void* SetKeysCtx;        /* Generate Sesssion Key Callback Context */
+    void* TlsFinishedCtx;    /* Generate Tls Finished Callback Context */
+    void* VerifymacCtx;      /* Verify mac Callback Context */
```

Comment:
Like above, this would be `VerifyMacCtx`

Reply:
Changed